hypothesis-python/setup.py

```diff
@@ -109,7 +106,6 @@ def local_file(name):
         "Operating System :: POSIX",
         "Operating System :: Microsoft :: Windows",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 2.7",
```

Comment:
Dunno how much you care, but I'm pretty sure there's a "Python 3 only" trove classifier we can add now.